operators/pkg/instance-controller/container_logic.go

```diff
@@ -143,7 +143,7 @@ func buildContainerInstanceDeploymentSpec(
 			}},
 			Env: []v1.EnvVar{{
 				Name:  "HIDE_NOVNC_BAR",
-				Value: strconv.FormatBool(examMode),
+				Value: strconv.FormatBool(environment.Mode == clv1alpha2.ModeExam || environment.Mode == clv1alpha2.ModeExercise),
```

Comment:
```suggestion
				Value: strconv.FormatBool(environment.Mode != clv1alpha2.ModeStandard)
```

Nit. Just a bit shorter.

Reply:
It is, however I preferred this to be explicitly done in case of those two modes only. Can change it since it does the same thing functionally by now

Reply:
As you prefer. Since in the other cases you always checked for `ModeStandard` it should be fine also here.